api/Cargo.toml

```diff
@@ -21,6 +21,7 @@ http-types = "2.11"
 json-patch = "0.2"
 password-hash = "0.2"
 rand = "0.8"
+reqwest = { version = "0.11", features = ["json", "blocking"] }
```

Comment:
we need to use async reqwest

Reply:
actually I just checked and surf supports http2, we need to figure out what's going on instead of using reqwest